controllers/usersController.js

```diff
@@ -262,6 +262,44 @@ class UserController {
       });
     }
   }
+
+  static async changePassword(req, res) {
+    try {
+      const { newPassword, currentPassword } = req.body;
+      const usernameFromToken = req.user.username;
+      const { username } = req.params;
+      const salt = await genSaltSync(
+        parseFloat(process.env.BCRYPT_HASH_ROUNDS) || 10
+      );
+      const response = await User.findOne({ where: { username } });
```

Comment:
What do you think by finding a user by ID since it is quicker than finding by username?

Reply:
Hi @abayo-luc!
I also think we should use ID. But we have used a `username` to fetch users information across the whole application. It might require a lot of refactoring, maybe we should change to ID in the future. Thanks.

Reply:
cool